src/components/DatePickerModal/readme.md

```diff
@@ -1,72 +1,381 @@
+    let dates = getDate(state.value);
+    if (dates) {
+        if (dates.length > 1) {
+            startDate =
+                (dates[0].getMonth() + 1).toString().padStart(2, '0') +
+                '/' +
+                dates[0]
+                    .getDate()
+                    .toString()
+                    .padStart(2, '0') +
+                '/' +
+                dates[0].getFullYear();
+            endDate =
+                (dates[1].getMonth() + 1).toString().padStart(2, '0') +
+                '/' +
+                dates[1]
+                    .getDate()
+                    .toString()
+                    .padStart(2, '0') +
+                '/' +
+                dates[1].getFullYear();
+            rangeDate = startDate + ' - ' + endDate;
+            setState({ dates: rangeDate });
+        } else {
+            rangeDate =
+                (dates[0].getMonth() + 1).toString().padStart(2, '0') +
+                '/' +
+                dates[0]
+                    .getDate()
+                    .toString()
+                    .padStart(2, '0') +
+                '/' +
+                dates[0].getFullYear();
+            setState({ dates: rangeDate });
+        }
+    }
+    setState({ isOpen: false });
+};
+
+<div className="rainbow-p-around_large">
+    <Row className="rainbow-border-radius_oval">
+        <div className="rainbow-flex rainbow-justify_space-around">
+            <Column className="rainbow-flex rainbow-justify_spread">
+                <div>
+                    <Title>LOCATION</Title>
+                    <Features>{state.location}</Features>
+                </div>
+                <div>
+                    <ButtonMenu
+                        menuAlignment="right"
+                        menuSize="x-small"
+                        buttonVariant="base"
+                        icon={<ArrowIcon />}
+                    >
+                        <MenuItem
+                            label="Mexico"
+                            onClick={() => {
+                                setLocation('Mexico');
+                            }}
+                        />
+                        <MenuItem
+                            label="Brasil"
+                            onClick={() => {
+                                setLocation('Brasil');
+                            }}
+                        />
+                        <MenuItem
+                            label="Canada"
+                            onClick={() => {
+                                setLocation('Canada');
+                            }}
+                        />
+                        <MenuItem
+                            label="United State"
+                            onClick={() => {
+                                setLocation('United State');
+                            }}
+                        />
+                    </ButtonMenu>
+                </div>
+            </Column>
+            <Vl />
```

Comment:
let's avoid the use of abbreviations, we need to make the code more descriptive and readable